hw/top_earlgrey/data/ip/chip_edn_testplan.hjson

```diff
@@ -0,0 +1,122 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  name: chip_edn
+  testpoints: [
+    // EDN (pre-verified IP) integration tests:
+    {
+      name: chip_sw_edn_boot_mode
+      desc: '''Verify EDN operation in boot-time request mode.
+
+            Verify that EDN delivers entropy in boot-time request mode and that the `edn_fips` bit correctly signals to endpoints whether the delivered entropy is derived from a FIPS/CC compliant seed or not.
+
+            Procedure:
+            - Disable the entropy complex.
+            - Enable ENTROPY_SRC in boot-time mode, e.g., by setting CONF.FIPS_ENABLE to kMultiBitBool4False.
+            - Enable CSRNG.
+            - Enable EDN1 (the one connected to OTBN RND) in boot-time request mode.
+            - Enable EDN0 (the one connected to OTBN URND and other endpoints) in auto request mode.
+            - Launch an OTBN program consuming entropy via both the RND and the URND interface.
+            - Verify that the OTBN program completes (entropy available) and that OTBN signals a recoverable alert and sets ERR_BITS.RND_FIPS_CHK_FAIL to indicate the reception of non-FIPS/CC-compliant entropy over the RND interface.
+            - Disable the entropy complex.
+            - Enable ENTROPY_SRC in FIPS mode.
+            - Enable CSRNG.
+            - Enable EDN1 and EDN0 in auto request mode.
+            - Re-launch the OTBN program.
+            - Verify that the OTBN program completes (entropy available) and that OTBN does not signal a recoverable alert and does not set ERR_BITS.RND_FIPS_CHK_FAIL.
+            '''
+      features: [
+        EDN.MODE.BOOT,
+        EDN.MODE.AUTO,
+        EDN.TRACK_SEED_COMPLIANCE
+      ]
+      stage: V3
+      si_stage: SV2
+      lc_states: ["PROD"]
+      bazel: []
+    },
+    {
+      name: chip_sw_edn_auto_mode
+      desc: '''Verify EDN operation in auto request mode.
+
+            Verify that EDN delivers entropy in auto request mode.
+
+            Procedure:
+            - Disable the entropy complex.
+            - Enable ENTROPY_SRC in FIPS mode.
+            - Enable CSRNG.
+            - Enable EDN0 and EDN1 in auto request mode.
```

Comment:
Suggest adding "and with non-deterministic seeds (`flag0 = kMultiBitBool4True` for Instantiate and Reseed commands)" as otherwise alerts about deterministic / non-FIPS-compliant entropy may be raised

Reply:
Agreed and fixed as above.